_posts/2017-10-06-DevopsDays-London.md

```diff
@@ -0,0 +1,217 @@
+into eight rooms, and audience members are tasked with devising a topic
+and arguing for its inclusion. Each session lasts roughly 45 minutes per
+room, at which point a new topic begins). With speakers from The
+Guardian, The Financial Times and Pizza Hut, there was a wide range of
+opinions and viewpoints presented over the course of the two days.
+
+Having not visited a Devops conference for at least two years, I was
+excited to see how the industry was progressing; though I wasn’t
+disappointed, it was interesting to see how the rate of progress seemed
+to be slowing compared to a few years ago. As you might expect,
+there was still plenty of talk about containers and their place within
+the industry, but now the focus was on ‘what and where next’, and ‘is
+there life after Docker’ (the answer for both is, of course,
+Kubernetes). There was also interesting discussion on ‘keeping things
+boring’, and several talks on methods for implementing and standardizing
+Pipelines as Code. This seems to be a real pinch point for much of the
+community at present.
+
+Let’s have a look at some of the key topics covered in some more detail.
+
+Kubernetes
+==========
+
+The subject everybody was talking about. There were multiple strands
+around the subject – ‘where is future of the container industry?’, ‘what
+next for Docker?’, ‘why should I be using Kubernetes?’ – but a show of
+hands in a couple of the Open Spaces sessions showed that very few are
+using the technology in production (Kevin Bowman from Bet was one of the
+few participants to have in-depth knowledge of building and deploying
+Kubernetes in a live environment). On the whole these sessions had the
+early adopters extoling the benefits of the technology (network security
+policies, aggregated logging per cluster), and in particular its
+advantages over ECS and Docker Swarm (service discovery, intelligent
+routing and per node affinity rules). Some mention was also made of its
+inability to cope with ‘big data’ workloads – instead a combination of
+Mesos and Marathon was recommended for this.
+
+What can we take from this? Undoubtedly Kubernetes is an exciting
+prospect, but one that is still in its infancy. It was telling how few
+are currently using this for production workloads – perhaps ‘boring is
+powerful’ after all. Which leads us neatly on to…
+
+Boring is powerful
+==================
+
+Jon Topper from The Scale Factory gave a great talk on idea of keeping
+your platform ‘boring’, which can be split into two main ideas; first of
+all, that software is rarely complete right from the start, features are
+always added over time. As the ‘fashionableness’ of a product decreases
+over time, feature availability, quality, reliability, security and
+collective knowledge all increase. The release schedule of former
+industry darling MongoDB illustrates the point:
+
+![MongoDB release schedule](/images/MongoDB_release_schedule.png)
```

Comment:
this would be more accessible as a table, but that might not be critical if the point is just to show when data journalling was added

Reply:
I can't find a table copy of this I'm afraid.